lib/src/presentation/view/home_page.dart

```diff
@@ -18,20 +20,26 @@ class _HomePageState extends State<HomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(
-          widget.title,
+        title: Container(
+          alignment: Alignment.center,
+          child: Text(
+            widget.title,
+          ),
```

Comment:
you missed this alignment, change it like the others

Reply:
done